docs/manual/comparison_with_sql.rst

```diff
@@ -0,0 +1,662 @@
+               by('Fruit','Name')][f.sum_num > 50, :]
+
+.. csv-table:: Filtration after a Group By
+    :header: Fruit, Name, sum_num
+    :widths: 10,10, 10
+
+    Grapes,	Tom,	87
+	Oranges,	Bob,	67
+	Oranges,	Mike,	57
+
+
+- Grouping on a condition
+
+.. code-block:: SQL
+
+    SELECT sepal_width >=3 as width_larger_than_3,
+           AVG(sepal_length) as avg_sepal_length
+    FROM iris
+    GROUP BY sepal_width>=3;
+
+
+.. code-block:: python
+
+    # datatable
+    iris[:,
+         {"avg_sepal_length": dt.mean(f.sepal_length)},
+         by(f.sepal_width >= 3)]
+
+.. csv-table:: Grouping on a Condition
+    :header: CO, avg_sepal_length
+    :widths: 10,10
+
+    	0,	5.95263
+    	1,	5.77634
+
+At the moment, names cannot be assigned in the ``by`` section.
+
+LEFT OUTER JOIN
+----------------
+
+We will compare the left outer join, as that is the only join currently implemented in ``datatable``. Another aspect is that the dataframe being joined must be keyed, the column or columns to be keyed must not have duplicates, and the joining column has to have the same name in both dataframes. You can read more about the :func:`join()` API and have a look at the `Tutorial on the join operator <https://datatable.readthedocs.io/en/latest/start/quick-start.html#join>`_
+
+Example data ::
+
+    DT = dt.Frame(x = ["b"]*3 + ["a"]*3 + ["c"]*3,
+                  y = [1, 3, 6] * 3,
+                  v = range(1, 10))
+
+    X = dt.Frame({"x":('c','b'),
+                  "v":(8,7),
+                  "foo":(4,2)})
+
+A left outer join in SQL :
+
+.. code-block:: SQL
+
+    SELECT DT.x,
+           DT.y,
+           DT.v,
+           X.foo
+    FROM DT
+    left join X
+    on DT.x = X.x
+
+A left outer join in ``datatable`` :
+
+.. code-block:: python
+
+    X.key = 'x'
+    DT[:, [f.x, f.y, f.v, g.foo], join(X)]
+
+.. csv-table:: Left Outer Jion
+    :header:     x,	y,	v,	foo
+    :widths: 10,10,10,10
+
+	b,	1,	1,	2
+	b,	3,	2,	2
+	b,	6,	3,	2
+	a,	1,	4,	NA
+	a,	3,	5,	NA
+	a,	6,	6,	NA
+	c,	1,	7,	4
+	c,	3,	8,	4
+	c,	6,	9,	4
+
+UNION
+------
+
+The ``UNION ALL`` clause in SQL can be replicated in ``datatable`` with :func:`rbind()`.
+
+.. code-block:: SQL
+
+    SELECT x, v
+    FROM DT
+    UNION ALL
+    SELECT x, v
+    FROM x
+
+In ``datatable``, :func:`rbind()` takes a list/tuple of dataframes and lumps into one :
+
+.. code-block:: python
+
+    dt.rbind([DT[:, ('x','v')], X[:, ('x', 'v')]])
+
+.. csv-table:: Union all
+    :header: x, v
+    :widths: 10,10
+
+    	b,	1
+    	b,	2
+    	b,	3
+    	a,	4
+    	a,	5
+    	a,	6
+    	c,	7
+    	c,	8
+    	c,	9
+    	b,	7
+    	c,	8
+
+SQL's ``UNION`` removes duplicate rows after combining the results of the individual queries; there is no built-in function in ``datatable`` yet that handles duplicates.
+
+SQL's WINDOW FUNCTIONS
+----------------------
+
+Some SQL window functions can be replicated in datatable (`rank` is one of the windows function not currently implemented in datatable) :
```

Comment:
I would add back-quotes to `datatable` here to be consistent with the rest of the content.